pandas/_libs/tslibs/conversion.pyx

```diff
@@ -1130,21 +1130,11 @@ def normalize_i8_timestamps(int64_t[:] stamps, object tz=None):
     result : int64 ndarray of converted of normalized nanosecond timestamps
     """
     cdef:
-        Py_ssize_t i, n = len(stamps)
-        npy_datetimestruct dts
+        Py_ssize_t n = len(stamps)
         int64_t[:] result = np.empty(n, dtype=np.int64)
 
-    if tz is not None:
-        tz = maybe_get_tz(tz)
-        result = _normalize_local(stamps, tz)
-    else:
```

Comment:
Is this case never reached?

Reply:
Correct. This case (the naive case) is handled in these two places now:

https://github.com/pandas-dev/pandas/pull/23634/files#diff-231ac35d2116a12844a7cfed02730580R1289
https://github.com/pandas-dev/pandas/pull/23634/files#diff-960da60b33d858481ad8799b0e84764bR835